src/components/common/form/FormDatePicker.tsx

```diff
@@ -0,0 +1,46 @@
+import { TextField } from '@mui/material'
+import { LocalizationProvider, DatePicker } from '@mui/x-date-pickers'
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns'
+import { format } from 'date-fns'
+import { useField, useFormikContext } from 'formik'
+import { i18n } from 'next-i18next'
+
+import { DATE_VALUE_FORMAT, getDateFormat } from 'common/util/date'
+
+/**
+ * MUI date picker to be connected with Formik. Propagates updates to the passed Formik field name
+ * @param name - name of the Formik field to bind
+ * @param label - prompt text
+ * @returns
+ */
+export default function FormDatePicker({ name, label }: { name: string; label: string }) {
+  const [field] = useField(name)
+  const { setFieldValue } = useFormikContext()
+
+  const dateViewFormat = getDateFormat(i18n?.language ?? '')
```

Comment:
It's best to use the `i18n` object from the `useTranslation()` hook so the component is re-rendered once the language changes.

```tsx
const { i18n } = useTranslation()
```